src/main/java/tutorease/address/logic/commands/FindLessonCommand.java

```diff
@@ -0,0 +1,59 @@
+package tutorease.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import tutorease.address.commons.util.ToStringBuilder;
+import tutorease.address.logic.Messages;
+import tutorease.address.model.Model;
+import tutorease.address.model.lesson.LessonContainsNamesPredicate;
+
+/**
+ * Finds and lists all lessons in address book whose student's name contains any of the argument keywords.
+ * Keyword matching is case-insensitive.
+ */
+public class FindLessonCommand extends Command {
+
+    public static final String COMMAND_WORD = "find";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all lessons whose students' names "
+            + "match the specified names (case-insensitive) and displays them in the lesson panel. \n "
+            + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
+            + "Example: " + COMMAND_WORD + " alice bob charlie";
```

Comment:
Should the message usage be lesson find? If not we will have duplicates.

Reply:
Fixed